src/screens/HomeScreen.web.tsx

```diff
@@ -0,0 +1,68 @@
+import { BACA_APP_URL, BACA_DOCS_URL } from '@baca/constants'
+import { Box, Button, Center, Text, ScrollView } from '@baca/design-system'
+import { useScreenOptions, useTranslation, useViewportDimensions } from '@baca/hooks'
+import { draftImages } from '@baca/screens'
+import { useCallback } from 'react'
+import { Linking, Image, ImageSourcePropType, StyleSheet } from 'react-native'
+
+export const HomeScreen = () => {
+  const { t } = useTranslation()
+  const { viewportWidth, viewportHeight } = useViewportDimensions()
+
+  useScreenOptions({ title: t('navigation.screen_titles.home') })
+
+  const openLink = useCallback((url: string) => Linking.openURL(url), [])
+
+  const renderItem = (item: ImageSourcePropType, index: number) => (
+    <Box key={index} height={viewportHeight * 0.6} width={viewportWidth * 0.6}>
+      <Image source={item} resizeMode="contain" style={styles.imageSize} />
+    </Box>
+  )
+
+  return (
+    <Center flex={1} px={4}>
+      <Text.XxlBold textAlign="center">{t('home_screen.header_title')}</Text.XxlBold>
+      <Text.LgRegular textAlign="center">{t('home_screen.header_subtitle')}</Text.LgRegular>
+
+      <Box
+        alignItems={'center'}
+        flexDirection={'row'}
+        flexWrap={'wrap'}
+        justifyContent={'center'}
```

Comment:
It don't have any impact on code you can change all this props like this:

```suggestion
        justifyContent="center"
```